pygmt/__init__.py

```diff
@@ -19,8 +19,7 @@
 """
 
 import atexit as _atexit
-
-from pkg_resources import get_distribution
+from importlib.metadata import version
```

Comment:
So `pylint` is complaining about the `version` variable being redefined:

```
pylint pygmt setup.py doc/conf.py
************* Module pygmt
pygmt/__init__.py:137:16: W0621: Redefining name 'version' from outer scope (line 22) (redefined-outer-name)
pygmt/__init__.py:150:12: W0621: Redefining name 'version' from outer scope (line 22) (redefined-outer-name)
```

Maybe rename `version` to `gs_version` and `gmt_version` at L137/L140 and L150/L153 respectively.